packages/vanilla/src/basics/form-elements/dropdown/tests/tests-dropdown.html

```diff
@@ -87,114 +98,136 @@
-                "type": "h2"
-            });
-
-            TitleEventsDropdown.mount('.test-heading-events');
-
-            const EventsDropdown = new Dropdown();
-
-            EventsDropdown.mount(".test-mount-events");
-            EventsDropdown.setSelectedOptionLabel('Imperceptible');
-
-            function log(componentName, eventType) {
-                const message = componentName + " triggered an " + eventType + " event";
-                console.log(message);
-            }
-
-            EventsDropdown.onClickOutside(log.bind(this, 'Dropdown', 'onClickOutside'));
-            EventsDropdown.onBlur(log.bind(this, 'Dropdown', 'onBlur'));
-            EventsDropdown.onFocus(log.bind(this, 'Dropdown', 'onFocus'));
-            EventsDropdown.onTargetClick(log.bind(this, 'Dropdown', 'onTargetClick'));
-            EventsDropdown.onKeypress(log.bind(this, 'Dropdown', 'onKeypress'));
-
-            [
-                { label: 'Uncommitted', value: 'foo' },
-                { label: 'Imperceptible', value: 'bar', selected: true },
-                { label: 'Arguable', value: 'baz' },
-            ].forEach(function(optionData)  {
-                const option = new EventsDropdown.partials.Option(optionData);
-                EventsDropdown.addOption(option);
-                option.onClick(log.bind(this, "Option " + optionData.label,  'onClick'));
-                option.onHover(log.bind(this, "Option" + optionData.label, 'onHover'));
-            });
-            EventsDropdown.open();
+        // ----------------------------
+        // DISABLED DROPDOWN
+        // ----------------------------
 
-        </script>
+        const TitleDisabledDropdown = new Hig.Typography({
+            "text": "Disabled Dropdown",
+            "type": "h2"
+        });
+
+        TitleDisabledDropdown.mount('.test-heading-disabled');
+
+        const DisabledDropdown = new Dropdown({
+            label: "Can't touch this",
+        });
+
+        DisabledDropdown.mount(".test-mount-disabled");
+        DisabledDropdown.setSelectedOptionLabel('Vaping');
+        DisabledDropdown.disable();
+
+
+
+        // ----------------------------
+        // DROPDOWN CONFIGURED WITH SETTERS
+        // ----------------------------
+
+        const TitleSetterDropdown = new Hig.Typography({
+            "text": "Dropdown configured with setters",
+            "type": "h2"
+        });
+
+        TitleSetterDropdown.mount('.test-heading-setters');
+
+        const SetterDropdown = new Dropdown();
+
+        SetterDropdown.mount(".test-mount-setters");
+
+        [{
+                label: 'Uncommitted',
+                value: 'foo'
+            },
+            {
+                label: 'Imperceptible',
+                value: 'bar',
+                selected: true
+            },
+            {
+                label: 'Arguable',
+                value: 'baz'
+            },
+        ].forEach(function (optionData) {
+            const option = new Hig.Option();
```

Comment:
Dropdown example uses `Hig.Option` rather than `Dropdown._partials.Option` because the option component is shared between a few components now.